subiquity/server/controllers/tests/test_filesystem.py

```diff
@@ -127,3 +137,154 @@ def test_good_modes(self, mode):
     def test_bad_modes(self, mode):
         with self.assertRaises(ValueError):
             self.fsc.validate_layout_mode(mode)
+
+
+bootloaders = [(bl, ) for bl in list(Bootloader) if bl != Bootloader.NONE]
```

Comment:
Eh. Some of these tests should run in the NONE case surely?

Reply:
Uh right. Well guided partitioning should certainly work in the bl==None case! I guess can_be_boot_disk should always return True in this case and add_boot_disk should be a noop? (either by checks in those functions or get_boot_part_plan returning some kind of noop plan in the bl==None case?)

Reply:
Added `NoOpBootPlan`